plugins/frontend/backstage-plugin-cloudsmith/src/api/CloudsmithApi.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createApiRef } from '@backstage/core-plugin-api';
+
+export const cloudsmithApiRef = createApiRef<CloudsmithApi>({
```

Comment:
shouldn't this be createApiRef<CloudsmithClient>

Reply:
I'm not sure what you mean exactly.

Reply:
Missing the important part. 
I meant to write createApiRef<CloudsmithClient>

Reply:
I think it is correct already.